modules/ROOT/pages/indexes/search-performance-indexes/overview.adoc

```diff
@@ -0,0 +1,24 @@
+:description: Overview of the search-performance indexes available in Neo4j.
+= Search-performance indexes
+
+Search-performance indexes enable quicker retrieval of exact matches between an index and the primary data storage.
+There are four different search-performance indexes available in Neo4j:
+
+* *Range indexes*: Neo4j’s default index.
+Supports most types of predicates, including node label and relationship types.
+
+* *Text indexes*: solves predicates operating on `STRING` values.
+Optimized for queries filtering `STRING` properties for what they `CONTAIN` or `ENDS WITH`.
```

Comment:
```suggestion
Optimized for queries filtering with the `STRING` operators `STARTS WITH`, `CONTAINS`, and `ENDS WITH`.
```

Reply:
Your suggestion is incorrect for STARTS WITH (range indexes solve these much faster due to ordering) - but I will consider the other two :)